homeassistant/components/lametric/quality_scale.yaml

```diff
@@ -0,0 +1,75 @@
+rules:
+  # Bronze
+  action-setup: done
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
+  docs-actions: done
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: todo
+  entity-event-setup:
+    status: exempt
+    comment: |
+      Entities of this integration does not explicitly subscribe to events.
+  entity-unique-id: done
+  has-entity-name: done
+  runtime-data: todo
+  test-before-configure: done
+  test-before-setup: done
+  unique-config-entry: done
+
+  # Silver
+  action-exceptions: done
+  config-entry-unloading: done
+  docs-configuration-parameters: todo
+  docs-installation-parameters: todo
+  entity-unavailable: done
+  integration-owner: done
+  log-when-unavailable: done
+  parallel-updates: todo
+  reauthentication-flow: done
+  test-coverage: done
```

Comment:
Would it be nice if we could test `test_helpers` implicitly via the services so we actually assert how the services respond to an invalid device id or no coordinator found?

Reply:
Sorry, disagree.

Reply:
If I remove `test_helpers.py` we don't cover both `raise` statements of the `async_get_coordinator_by_device_id`, so no, we don't cover this multiple times

Reply:
It is covered with it being there, which complies with the IQS requirements.